Casks/stretchly.rb

```diff
@@ -1,12 +1,29 @@
 cask "stretchly" do
-  version "1.9.0"
-  sha256 "ca4f100f7291e5a0ddde06cec7295f843f6ae06df6551808c412f537eaaf2d15"
+  arch = Hardware::CPU.intel? ? "x64" : "arm64"
```

Comment:
Since it's only used in the URL, let's do
```suggestion
  arch = Hardware::CPU.intel? ? "" : "-arm64"
```